src/openrct2/interface/Screenshot.cpp

```diff
@@ -284,10 +284,9 @@ sint32 cmdline_for_gfxbench(const char **argv, sint32 argc)
 
     gOpenRCT2Headless = true;
     auto context = CreateContext();
-    if (context->Initialise())
+    if (context->Initialise() && context->LoadParkFromFile(inputPath))
     {
         drawing_engine_init();
-        context->LoadParkFromFile(inputPath);
```

Comment:
This may be silly but is it safe to load the park before the drawing engine is initialised? There shouldn't be independencies between them but thought it might be worth mentioning

Reply:
Hmm... it does call `sprite_position_tween_reset()`.

Reply:
I tested it like this with an park it couldn't find and with our bpb.sv6, got the same results as before. But yeah I'm not too happy about this but I wanted to avoid refactoring since the context needs to be deleted later. Could probably wrap the context in unique ptr.